.travis.yml

```diff
@@ -22,9 +22,11 @@ before_install:
   - npm i -g jshint
 
 script:
+  # Run coala before build.sh due to the issue
```

Comment:
do you know which files are causing it?
can you exclude those from the keyword search?
that would be better.

Reply:
This is not a problem anymore. I dumped `meta_review.json` to `static` folder, which should be in `_site` folder. It is fixed now.